resources/js/Pages/Dashboard.tsx

```diff
@@ -1,11 +1,41 @@
 import React, { ReactElement } from "react";
+import { Chart } from "../components/Chart";
+import { Panel } from "../components/Panel";
 import { AuthenticatedLayout } from "../layout/AuthenticatedLayout";
 
 const Dashboard: React.VFC = () => {
 	return (
-		<div className="px-4 py-6 sm:px-0">
-			<div className="border-4 border-dashed border-gray-200 rounded-lg h-96" />
-		</div>
+		<Panel
```

Comment:
I think this chart is supposed to go on `Reports` page. Probably for now its fine, but I think we should discuss what is supposed to go into dashboard and what to reports